src/goto-programs/builtin_functions.cpp

```diff
@@ -636,6 +636,88 @@ void goto_convertt::do_enum_is_in_range(
   copy(assignment, ASSIGN, dest);
 }
 
+void goto_convertt::do_havoc_slice(
+  const exprt &lhs,
+  const symbol_exprt &function,
+  const exprt::operandst &arguments,
+  goto_programt &dest)
+{
+  std::string identifier = CPROVER_PREFIX "havoc_slice";
```

Comment:
Use `irep_idt` instead of `std::string`.

Reply:
done